tools/ds-identify

```diff
@@ -319,6 +372,11 @@ detect_virt() {
                 *) virt="$out"
             esac
         }
+        out=$(sysctl -qn security.jail.jailed 2>/dev/null) && {
+            if [ "$out" = "1" ]; then
+                virt="jail"
+            fi
+        }
```

Comment:
I'm not sure if this shouldn't be part of a different changeset

Reply:
i'm fine with it here, but maybe call it bsd-json?

Reply:
I'm going by what other tools like `facter` return:

```
root@webirc:~ # facter virtual
jail
root@webirc:~ #
```

on a completely unrelated note: while it's possible to run Linux in a FreeBSD jail, the version is so outdated, that our tools would barely pick run on it, so, :woman_shrugging: